helloworld/lib/goodbye.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Are the goodbye modules yours or Duncan's example code? I assume you just had them here for your own reference. That's totally cool to keep these modules on here, but it might be best to add them in your .gitignore so they don't get pushed up to your repo.

I think you just do do that by adding lib/goodbye.js and whatever other files to your .gitignore, but might be good to research that for curiosity.
https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#Ignoring-Files